legacyCreateAccountFragment/src/main/java/com/hamdan/forzenbook/legacy/createaccount/fragment/LegacyCreateAccountFragment.kt

```diff
@@ -66,115 +68,124 @@ class LegacyCreateAccountFragment : Fragment() {
-                                            getString(R.string.create_account_first_name_error_invalid_characters)
-                                        isVisible = true
+                                    firstNameErrorText.apply {
+                                        if (stateContent.firstName.text.isNotEmpty() && !stateContent.firstName.error.isValid()) {
+                                            if (stateContent.firstName.error == EntryError.NameError.Length) {
+                                                text =
+                                                    getString(R.string.create_account_first_name_error_length)
+                                                isVisible = true
+                                            } else if (stateContent.firstName.error == EntryError.NameError.InvalidCharacters) {
+                                                text =
+                                                    getString(R.string.create_account_first_name_error_invalid_characters)
+                                                isVisible = true
+                                            }
+                                        } else {
+                                            isVisible = false
+                                        }
                                     }
-                                } else {
-                                    isVisible = false
-                                }
-                            }
 
-                            lastNameErrorText.apply {
-                                if (lastName.text.isNotEmpty() && !lastName.error.isValid()) {
-                                    if (lastName.error == LoginError.NameError.Length) {
-                                        text =
-                                            getString(R.string.create_account_last_name_error_length)
-                                        isVisible = true
-                                    } else if (lastName.error == LoginError.NameError.InvalidCharacters) {
-                                        text =
-                                            getString(R.string.create_account_last_name_error_invalid_characters)
-                                        isVisible = true
+                                    lastNameErrorText.apply {
+                                        if (stateContent.lastName.text.isNotEmpty() && !stateContent.lastName.error.isValid()) {
+                                            if (stateContent.lastName.error == EntryError.NameError.Length) {
+                                                text =
+                                                    getString(R.string.create_account_last_name_error_length)
+                                                isVisible = true
+                                            } else if (stateContent.lastName.error == EntryError.NameError.InvalidCharacters) {
+                                                text =
```

Comment:
What the formatting lol

Reply:
ye, I know, its awkward because its so far out to the right that the autoformatter always pushes it to the next line.

Reply:
I could manually move it but anytime the file is formatted itll be moved back down to the next line.